src/ray/gcs/gcs_client/global_state_accessor.cc

```diff
@@ -174,6 +174,15 @@ std::string GlobalStateAccessor::GetInternalConfig() {
   return config_proto.SerializeAsString();
 }
 
+std::vector<std::string> GlobalStateAccessor::GetAllHeartbeat() {
```

Comment:
How about add a test  to global_state_accessor_test.cc?

Reply:
sure